### ISSUE TITLE
Durham County Council

### ISSUE THREAD

**Issue by Mod74** [robbrad/UKBinCollectionData#3]
Could you try Durham CC?

They seem to provide a unique page for each house, for example

http://mydurham.durham.gov.uk/article/12690?uprn=10091173263

(UPRN, Unique Property Reference Number)

Which has the bin collections as text at the top.

Thanks!

**Comment by robbrad** [comment]
Done : https://github.com/robbrad/UKBinCollectionData/commit/f0a7c3dce91b0d7ffeb9da97bd7088125f53684e

**Comment by Mod74** [comment]
Brilliant, thank you!

**Comment by Mod74** [comment]
Hi, I think I have this working from a Home Assistant / Python point of view, however when I run I get a Python Syntax error

Syntax errors detected :

Line 26:
data[f'{binType}'] = f'{binCollection}'
^
SyntaxError: invalid syntax

I know next to nothing about coding so I tried checking here https://extendsclass.com/python-tester.html and it shows the same error. I checked the other bin collection scripts and the one that is similar to this throws the same error, but the ones that are completely different doesn't. I checked pythonbuddy.com and it said "Format strings are only supported in Python 3.6 and greater (, line 26)" however I have 3.7 installed as Home Assistant required it and the venv was set up using it. I tried to shebang the specific version 3.7.3 and it still threw the syntax error. Any idea?
